### PR TITLE
Add helper script to fetch all sources of a given release

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Could be useful in the future :shrug:

**Review by tranchitella — APPROVED** [review]
Thanks, nifty!

File: extra/get-release-source.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    wget --header "Authorization: token ${GITHUB_TOKEN}" ${url}
+    mv ${repo_version} mender-${RELEASE_VERSION}-src/${repo}-${repo_version}.tar.gz
+done
+
+echo
+echo "Donwloaded source tarballs:"
+ls -1 mender-${RELEASE_VERSION}-src
+
+echo
+read -p "Remove source for OS repos that have Enteprrise fork? " -n 1 -r
```
Comment (kacf) [inline extra/get-release-source.sh:36]: Enteprrise

**Review by kacf — APPROVED** [review]
:+1:

**Pull request merged**